distributions/aws/examples/storage-efs/README.md

```diff
@@ -0,0 +1,153 @@
+
+2. At this point, you have likely cloned this repo and checked out the right branch. Navigate over this this directory. 
+
+3. Make sure the following environment variables are set. 
+```
+export CLUSTER_NAME=<clustername>
+export CLUSTER_REGION=<clusterregion>
+export AWS_ACCOUNT_ID=$(aws sts get-caller-identity --query "Account" --output text)
+```
+
+## 2.0 Install the EFS CSI Driver
+We recommend installing the EFS CSI Driver v1.3.4 directly from the [the aws-efs-csi-driver github repo](https://github.com/kubernetes-sigs/aws-efs-csi-driver) as follows - 
+
+```
+kubectl apply -k "github.com/kubernetes-sigs/aws-efs-csi-driver/deploy/kubernetes/overlays/stable/?ref=tags/v1.3.4"
+```
+
+You can confirm that EFS CSI Driver was installed into the default kube-system namespace for you. You can check using the following command - 
+```
+kubectl get csidriver
+
+NAME              ATTACHREQUIRED   PODINFOONMOUNT   MODES        AGE
+efs.csi.aws.com   false            false            Persistent   5d17h
+```
+
+## 3.0 Create the IAM Policy for the CSI Driver
+The CSI driver's service account (created during installation) requires IAM permission to make calls to AWS APIs on your behalf. Here, we will be annotating the Service Account `efs-csi-controller-sa` with an IAM Role which has the required permissions.
+
+1. Download the IAM policy document from GitHub as follows - 
+
+```
+curl -o iam-policy-example.json https://raw.githubusercontent.com/kubernetes-sigs/aws-efs-csi-driver/v1.3.4/docs/iam-policy-example.json
+```
+
+2. Create the policy - 
+```
+aws iam create-policy \
+    --policy-name AmazonEKS_EFS_CSI_Driver_Policy \
+    --policy-document file://iam-policy-example.json
+```
+
+3. Create an IAM role and attach the IAM policy to it. Annotate the Kubernetes service account with the IAM role ARN and the IAM role with the Kubernetes service account name. You can create the role using eksctl as follows - 
+
+```
+eksctl create iamserviceaccount \
+    --name efs-csi-controller-sa \
+    --namespace kube-system \
+    --cluster $CLUSTER_NAME \
+    --attach-policy-arn arn:aws:iam::$AWS_ACCOUNT_ID:policy/AmazonEKS_EFS_CSI_Driver_Policy \
+    --approve \
+    --override-existing-serviceaccounts \
+    --region $CLUSTER_REGION
+```
+
+4. You can verify by describing the specified service account to check if it has been correctly annotated - 
+```
+kubectl describe -n kube-system serviceaccount efs-csi-controller-sa
+```
+
+## 4.0 Create an Instance of the EFS Filesystem
+Please refer to the official [AWS EFS CSI Document](https://docs.aws.amazon.com/eks/latest/userguide/efs-csi.html#efs-create-filesystem) for detailed instructions on creating an EFS filesystem. 
+
+
+## 5.0 Using EFS Storage in Kubeflow
+
+## 5.1 Provisioning Options
+### Option 1 - Static Provisioning
```

Comment:
Where is the option 2 for dynamic provisioning ?

Reply:
I mentioned in standup and on this PR that I was planning to add that in a separate PR since this one has been long pending and that needs additional testing so I did not want to add to the scope.